front-end/src/styles/uploadModalStyles.css

```diff
@@ -82,4 +82,11 @@
   background: crimson;
   color: white;
   cursor: pointer;
+}
+
+.fileButton{
+  color: white;
+  background-color: red;
+  margin: 0;
+  padding: 7px 7px;
```

Comment:
What is this?

Reply:
its the x button next to the files that you upload, without it the button is massive and looks super dumb